src/js/scripts/utils.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-restricted-syntax */
+import fs from "fs";
+import fsPromises from "fs/promises";
+import path from "path";
+
+export async function walkDir(dir: string, callback: (itemPath: string) => void | Promise<void>) {
```

Comment:
Should we have a folder utils and isolate all utils that do not depend on anything there?

`js/utils/filesystem.ts`

Reply:
Done